external/os-autoinst-common/README.md

```diff
@@ -61,7 +61,7 @@ them and then do:
 
 You can find more information here:
 * [Repository and usage](https://github.com/ingydotnet/git-subrepo)
-* [A good comparison between subrepo, submodule and
+* [A good comparison beween subrepo, submodule and
```

Comment:
Unfortunately this change is re-introducing a typo (I've likely been fixing previously using codespell).

Reply:
ok, https://github.com/os-autoinst/os-autoinst-common/pull/9